app/src/main/res/values/strings.xml

```diff
@@ -3,17 +3,17 @@
     <string name="action_search">Search</string>
     <string name="action_add">Add</string>
     <plurals name="selectedCardCount">
-        <item quantity="one"><xliff:g>%d</xliff:g> card selected</item>
-        <item quantity="other"><xliff:g>%d</xliff:g> cards selected</item>
+        <item quantity="one"><xliff:g>%d</xliff:g> selected</item>
+        <item quantity="other"><xliff:g>%d</xliff:g> selected</item>
     </plurals>
-    <string name="noGiftCards">Click the + plus button to add a card, or import some from the ⋮ menu first.</string>
-    <string name="noGiftCardsGroup">You don\'t have any loyalty cards yet. Once you\'ve added some you can assign them to the group here.</string>
-    <string name="noMatchingGiftCards">Didn\'t find anything. Try changing your search.</string>
+    <string name="noGiftCards">Click the + plus button to make an entry, or import something from the ⋮ menu first.</string>
```

Comment:
I know that you can use the app for more than cards, but loyalty cards are the primary use case. I've already ran into people confused what to do, making the text less explicit really isn't something I want.